tracer/test/test-applications/integrations/Samples.AWS.SimpleNotificationService/SyncHelpers.cs

```diff
@@ -0,0 +1,91 @@
+#if NETFRAMEWORK
```

Comment:
why only for framework?

Reply:
Good question to ask for the .NET team, but in general. AFAIK, we test synchronous methods for .NET Framework, while Async methods are tested in .NET Core.